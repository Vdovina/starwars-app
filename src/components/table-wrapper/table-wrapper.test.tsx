import { create, act as testAct, ReactTestRenderer } from 'react-test-renderer';
import { TableWrapper } from './';


describe('TableWrapper Tests', () => {
  it('render', () => {
    let component: ReactTestRenderer;
    testAct(() => {
      component = create(<TableWrapper>ChildrenNode</TableWrapper>);
    });
    expect(component.toJSON()).toMatchSnapshot();
  });
});
