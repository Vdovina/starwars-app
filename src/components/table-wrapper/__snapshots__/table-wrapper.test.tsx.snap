// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableWrapper Tests render 1`] = `
<div
  className="table-wrapper"
>
  ChildrenNode
</div>
`;
